NuRadioMC/test/SingleEvents/T03validate.py

```diff
@@ -17,8 +17,7 @@
 fin1 = h5py.File(file1, 'r')
 fin2 = h5py.File(file2, 'r')
 
-attributes = [u'trigger_names',
```

Comment:
I think @cg-laser is working on changing all dicts to ordered dicts. Then we should be able to test the triggers again, right?